streamx-console/streamx-console-service/src/main/java/com/streamxhub/streamx/console/base/config/MybatisPlusConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.streamxhub.streamx.console.base.config;
```

Comment:
miss copyright header, before submit, you can run: mvn checkstyle:check

Reply:
Okay,Thank U